### ISSUE TITLE
[Playlist] Not able to add an audio file to the Playlist by tapping and holding on a file

### ISSUE THREAD

**Issue by hffvld** [brave/brave-ios#7667]
### Description: 

Not able to add an audio file to the Playlist by tapping and holding on a file

### Steps to Reproduce 
  1. Launch Brave
  2. Open www.music.youtube.com
  3. Play any audio track > Tap and hold on a file similar to how you do with a video file > Observe

### Actual result: 

There is no option to add an audio track by tapping and holding on a file. It's working for videos only.

**VIdeo**

https://github.com/brave/brave-ios/assets/128532965/f34c1062-4c81-487f-bba2-b5a1e630ace3

**Audio**

https://github.com/brave/brave-ios/assets/128532965/c96c0449-b781-4909-8b67-c66fbd0b6e01

### Expected result:

The user must be able to save the audio file to the Playlist by tapping and holding. The same way is working for the videos.

### Reproduces how often: [Easily reproduced, Intermittent Issue]

Easily reproduced

### Brave Version: 

Brave build: 1.52 (23.6.28.13)
BraveCore: 1.52.129 (114.0.5735.198)

- Can you reproduce this issue with the most recent build from TestFlight? Yes
- Can you reproduce this issue with the previous version of the current build from TestFlight? Yes
- Can you reproduce this issue with the current build from AppStore? Yes

### Device details: 

iPhone 14 [iOS 16.5]

### Website problems only:
- did you check with Brave Shields down? 
- did you check in Safari/Firefox (WkWebView-based browsers)?

**Comment by Brandon-T** [comment]
Technically, this is not a bug.
Audio elements don't have a `width` or `height`. What you're seeing there is a picture/thumbnail and long-pressing on it.
In other words, you're not actually touching the audio element. You're just touching the album's artwork.

In the case of Youtube though, it's also not an audio element. It's actually a `video` but with `width = 0` and `height = 0` so technically it's a video without a renderer that plays only audio lol.

In other words, this isn't supported and isn't necessary to fix because it's not actually a bug (it's intended that you can only long-press on `video` elements). However, I've added a PR that will fix it anyway.

**Comment by hffvld** [comment]
Verified on `iPhone 14` using version(s):
```
Device/OS: iPhone 14 [iOS 16.5.1]
Brave build: 1.56 (23.7.18.21)
BraveCore: 1.56.6 (115.0.5790.90)
```
---------------------------

**STEPS:**
  1. Launch Brave
  2. Open www.music.youtube.com
  3. Play any audio track > Tap and hold on a file similar to how you do with a video file > Observe

**ACTUAL RESULTS:**
- Verified that a user is able to save the audio file to the Playlist by tapping and holding. The same way is working for the videos.

---------------------------

https://github.com/brave/brave-ios/assets/128532965/fada23c4-5e49-48d1-bc17-bd50b8bb902f
